test-adapter/src/main/scala/org/scalajs/testing/adapter/TestAdapter.scala

```diff
@@ -167,4 +187,8 @@ object TestAdapter {
       val com: RPCCore,
       val mux: RunMuxRPC
   )
+
+  private abstract class ThreadIDAccessor {
+    def getCurrentThreadId(): Long
```

Comment:
I'm wondering if it would make since to make this `threadId` so it matches what you would call if you supported only JDK 19+? Technically it is `currentThreadId` as it could be modified but ...

Reply:
It's `currentThread()`+`threadId()`, so `(get)CurrentThreadId()`.

Unless I misunderstood your comment?

Reply:
@ekrich do you feel this needs follow-up? Otherwise I think this is ready for merging.

Reply:
It was totally readable. I just thought since it is calling `threadId()` that is should be named the same. Later when older JVMs are dropped the wrapper would just be removed and the name would remain the same.